api/pom.xml

```diff
@@ -186,23 +186,13 @@
       <artifactId>guava</artifactId>
       <version>${guava.version}</version>
     </dependency>
+    <!-- https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter-engine -->
     <dependency>
       <groupId>org.junit.jupiter</groupId>
-      <artifactId>junit-jupiter-params</artifactId>
+      <artifactId>junit-jupiter-engine</artifactId>
```

Comment:
I think this dependency can be removed altogether

Reply:
...looks like you're right :) https://github.com/bcgov/EDUC-EDX-API/pull/214/commits/bf76d207ba3e1abc1f0be7f66497e7b6c1ae293d